admin-node-setup.sh

```diff
@@ -39,6 +39,12 @@ for i in $(ls $images_dir/sles*.tag $images_dir/kubic*.tag);do
 
     echo "$0: Setting $image_name:$tag into manifests"
     sed -i -e "s%$image_name:__TAG__%$image_name:$tag%g" $tmp_dir/*.yaml
+
+    if [[ $image_name =~ pause ]]; then
+      kubelet_config=/etc/kubernetes/kubelet
+      echo "$0: Setting $image_name:$tag into kubelet config"
+      sed -i -e "s%$image_name:__TAG__%$image_name:$tag%g" $kubelet_config
```

Comment:
This will only replace the tag once, which is no worse than before, but still will only happen once. Something like this maybe?

    sed -i -e "s| --pod-infra-container-image=.* | --pod-infra-container-image=$DIST/pause:__TAG__ |" /etc/kubernetes/kubelet

This would also allow removing any of the pause stuff from activate.sh - best to keep all in 1 place.